### PR TITLE
[CLEANUP] Migration HowTo #infra

### PR DESCRIPTION
**Pull request by xerc**

### CODE REVIEW

File: docs/get-started/migrating-from-sequel-pro.md [hunk 1]
```diff
@@ -7,43 +7,36 @@
...
-~/Library/Preferences/com.sequelpro.SequelPro.plist
-to
-~/Library/Containers/com.sequel-ace.sequel-ace/Data/Library/Preferences/com.sequel-ace.sequel-ace.plist
+2. Quit both `Sequel Pro` and `Sequel Ace`
 
-5. Copy
-~/Library/Application\ Support/Sequel\ Pro/Data/Favorites.plist
-to
-~/Library/Containers/com.sequel-ace.sequel-ace/Data/Library/Application\ Support/Sequel\ Ace/Data/Favorites.plist
+3. Copy Settings via `cp ~/Library/Preferences/com.sequelpro.SequelPro.plist ~/Library/Containers/com.sequel-ace.sequel-ace/Data/Library/Preferences/com.sequel-ace.sequel-ace.plist1`
```
Comment (Jason-Morcos) [inline docs/get-started/migrating-from-sequel-pro.md:21]: Looks like there was a 1 accidentally added to the end of this line?

**Review by Jason-Morcos — CHANGES_REQUESTED** [review]
Thanks for the improvements here! Just one little note

File: docs/get-started/migrating-from-sequel-pro.md (hunk 1 above)
Comment (xerc) [reply docs/get-started/migrating-from-sequel-pro.md:21]: true ; good catch

**Review by Jason-Morcos — APPROVED** [review]
Great! Thank you!

**Pull request merged**